src/Prefix.jl

```diff
@@ -109,12 +109,13 @@ function includedir(prefix::Prefix)
 end
 
 """
-    logdir(prefix::Prefix)
+    logdir(prefix::Prefix; subdir::AbstractString="")
 
-Returns the logs directory for the given `prefix`.
+Returns the logs directory for the given `prefix`.  If `subdir` is a non-empty string, that
+directory it is appended to the logdir of the given `prefix`.
 """
-function logdir(prefix::Prefix)
-    return joinpath(prefix, "logs")
+function logdir(prefix::Prefix; subdir::AbstractString="")
+    return strip_backslash(joinpath(prefix, "logs", subdir))
```

Comment:
Hilariously, `strip_backslash()` actually strips forward-slashes on Linux/macOS.

Reply:
I think it's a misnomer :facepalm: